packages/storefront-lwc/__mocks__/apollo-client.js

```diff
@@ -0,0 +1,81 @@
+import { register, ValueChangedEvent } from '@lwc/wire-service';
```

Comment:
This is a mock for the apollo-client wire adapter. In the future it could be made more generic to mock any wire adapter.